### DISCUSSION TITLE
cannt connect to the software

### DISCUSSION THREAD

**Discussion by ZANYANBU** [openbookings26/ContaCam#26] [Q&A]
My usb camera is detected in my computer  but not with Contacam? please help me

**Comment by openbookings26** [comment]
Can you provide the exact model of your USB camera? I've used a USB logitec camera in the past with no issues.
As a last resort, there's also software out there that will turn your USB camera into a network camera and provide you with an IP stream to use with Contacam. I've used that in the past too.
